tests/sentry/dynamic_sampling/rules/biases/test_ignore_health_checks_bias.py

```diff
@@ -1,42 +1,28 @@
-from unittest.mock import MagicMock, patch
-
 import pytest
 
 from sentry.dynamic_sampling.rules.biases.ignore_health_checks_bias import (
-    IgnoreHealthChecksRulesGenerator,
+    HEALTH_CHECK_GLOBS,
+    IgnoreHealthChecksBias,
 )
 
 
 @pytest.mark.django_db
-@patch(
-    "sentry.dynamic_sampling.rules.biases.ignore_health_checks_bias.IgnoreHealthChecksDataProvider"
-)
-def test_generate_bias_rules_v2(data_provider, default_project):
-    rule_id = 1001
-    sample_rate = 1.0
-    health_check_globs = ["/health", "/healtz", "/*health*"]
-
-    data_provider.get_bias_data.return_value = {
-        "id": rule_id,
-        "sampleRate": sample_rate,
-        "healthCheckGlobs": health_check_globs,
-    }
-
-    rules = IgnoreHealthChecksRulesGenerator(data_provider).generate_bias_rules(MagicMock())
+def test_generate_bias_rules_v2(default_project):
+    rules = IgnoreHealthChecksBias().generate_rules(project=default_project, base_sample_rate=1.0)
     assert rules == [
         {
             "condition": {
                 "inner": [
                     {
                         "name": "event.transaction",
                         "op": "glob",
-                        "value": health_check_globs,
+                        "value": HEALTH_CHECK_GLOBS,
                     }
                 ],
                 "op": "or",
             },
-            "id": rule_id,
-            "samplingValue": {"type": "sampleRate", "value": 1.0},
+            "id": 1002,
+            "samplingValue": {"type": "sampleRate", "value": 0.2},
```

Comment:
This change is unclear; why do we change value here?

Reply:
During the refactoring of some tests, I decided to change sample rates, in order to make it apparent that they can be different than 1.0.